### PR TITLE
Internationalization

### PR DESCRIPTION
**Pull request by TomDijkema**
Pull Request adds the i18n-next library to setup multi lingual support.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "dotenv-cli": "^6.0.0",
     "formik": "^2.2.9",
     "html-react-parser": "^3.0.4",
+    "i18next": "^22.4.10",
```
Comment (samleeflang) [inline package.json:26]: Do you need them both? `i18next` and `react-i18next`

**Review by samleeflang — APPROVED** [review]
Already spoken about approach. Looks good, just one small quesion

File: package.json (hunk 1 above)
Comment (TomDijkema) [reply package.json:26]: I used both. According to their documentation i18n-next is the core library that sets up the language support. The extension for React adds a hook method so the translate function (t) can easily be called on in components to use the translate functionality.

Comment (TomDijkema) [reply package.json:26]: Probably could do without, but should be more optimized.

Comment (samleeflang) [reply package.json:26]: Ok clear

**Pull request merged**